apps/token/src/routes/proposals/propose/update-market/propose-update-market.tsx

```diff
@@ -145,176 +147,182 @@ export const ProposeUpdateMarket = () => {
           isEnactmentDeadlineAtMaximum
         ),
       },
-    });
+    };
+  };
+
+  const onSubmit = async (fields: UpdateMarketProposalFormFields) => {
+    await submit(assembleProposal(fields));
+  };
+
+  const viewJson = () => {
+    const formData = watch();
+    downloadJson(
+      JSON.stringify(assembleProposal(formData)),
+      'vega-update-market-proposal'
+    );
   };
 
   return (
     <AsyncRenderer
       loading={networkParamsLoading && marketsLoading}
       error={networkParamsError && marketsError}
-      data={params && marketsData}
-    >
-      <Heading title={t('UpdateMarketProposal')} />
-      <VegaWalletContainer>
-        {() => (
-          <>
-            <ProposalMinRequirements
-              minProposalBalance={
-                params.governance_proposal_updateMarket_minProposerBalance
-              }
-              spamProtectionMin={params.spam_protection_proposal_min_tokens}
-              userAction={ProposalUserAction.CREATE}
-            />
+      data={{ ...params, ...marketsData }}
```

Comment:
follow up (non-blocking): can we just check quickly that none of the rest of the forms are likely to experience this issue?

Reply:
no worries, all the rest simply take `params`